cli/tsc/99_main_compiler.js

```diff
@@ -41,6 +41,13 @@ delete Object.prototype.__proto__;
     "listen",
     "listenDatagram",
     "openKv",
+    "connectQuic",
+    "listenQuic",
+    "QuicBidirectionalStream",
+    "QuicConn",
+    "QuicListener",
+    "QuicReceiveStream",
+    "QuicSendStream",
```

Comment:
Nitpick - add these to `runtime/fmt_errors.rs` `get_suggestions_for_terminal_errors` for better DX?